src/components/item_page/Item.js

```diff
@@ -1,33 +1,47 @@
 import React, { useState, useEffect } from 'react';
 import { ItemPage } from './ItemPage';
 import ItemBids from 'components/item_bids/ItemBids';
-import { dayEndedUTCString } from 'utilities/date';
+import { ISOString } from 'utilities/date';
 import { convertDollarsToPennies } from 'utilities/price';
 
 const Item = (props) => {
   const [item, setItem] = useState({});
   const [error, setError] = useState(null);
   const [refetch, setRefetch] = useState(false);
+  const [bidSuccess, setBidSuccess] = useState(false);
   const { uuid, user_uuid } = props.match.params;
 
   useEffect(() => {
     const fetchItems = async () => {
       const { data, errors } = await props.apiHandler.get(`/api/users/${user_uuid}/items/${uuid}`);
-      data ? setItem(data) : setError(errors);
+      if (data) {
```

Comment:
Why'd you move from a ternary to `if/else`?

Reply:
I thought I would need to add more conditions, but ended up not using it. I'll switch back to ternary